analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -85,25 +94,37 @@ class SpdxDocumentFile(
 
     /**
      * Return the dependencies of [pkg] defined in [doc] of the [SpdxRelationship.Type.DEPENDENCY_OF] type.
+     * ORT [Package]s are mapped during the identification, using [workingDir] to define the version control system
+     * directory of the dependencies and added to [packages].
```

Comment:
This sentence does not sound grammatically correct to me. Maybe rather something like:
```
Identified dependencies are mapped to ORT [Package]s and are added to [packages], using [workingDir] to define the
version control system directory of the dependencies.
```